src/components/Layout.tsx

```diff
@@ -16,14 +19,17 @@ interface Props {
 function Layout({ children }: Props): JSX.Element {
   // Hook to manage side-navigation drawer
   const { isOpen, onOpen, onClose } = useDisclosure();
+  const [locale, setLocale] = useLocalStorage<Locale>('locale', 'en-US');
```

Comment:
Do we also need to handle the cases where local storage is disabled on the client-side? For instance, show a warning message or store it in some React state?

Reply:
Eh, I think falling back to English is fine